docs/getting-started.md

```diff
@@ -216,91 +193,58 @@ The setup so far will automatically assert the Lighthouse team's recommended set
 lhci assert --rc-file=lighthouserc.json
 EXIT_CODE=$?
-
-# ...
 ```
 
-## Extra Goodies
+## Storing Reports
 
-### Saving Reports to Public Storage
+You have two options for storing the collected Lighthouse reports for later viewing.
 
-The existing setup will fail builds, print failing audits, and keep your project in line, but what happens when something goes wrong? You want to see the Lighthouse report!
+### Saving Reports to Temporary Public Storage
 
-There's a free service that provides temporary public storage of your Lighthouse reports so you can examine the HTML report of any build without tokens or extra infrastructure. Just add a single `lhci upload` command _after_ `lhci assert`.
+While the Lighthouse CI server offers the complete LHCI experience, you do have to host it yourself. If you'd like basic storage so you can just get a URL for each report, we offer a free service that provides temporary public storage.
 
-**NOTE: as the name implies this is _temporary_ and _public_ storage, do not use if you're uncomfortable with the idea of your Lighthouse reports being stored on a public URL on Google Infrastructure. Reports are automatically deleted 7 days after upload.**
+**NOTE: as the name implies this is _temporary_ and _public_ storage, do not use if you're uncomfortable with the idea of your Lighthouse reports being stored on a public URL on Google Cloud Infrastructure. Reports are automatically deleted 7 days after upload.**
 
 ```bash
-#!/bin/bash
-
-# ...
-
-lhci assert --rc-file=lighthouserc.json
-EXIT_CODE=$?
-
+# If you use `lhci assert`, upload must follow after it.
 lhci upload --target=temporary-public-storage
-
-# ...
 ```
 
 If you're using `autorun`, setting `ci.upload.target` to `temporary-public-storage` in your `lighthouserc.json` is all that's necessary. The upload step will happen automatically.
 
-### Historical Reports & Diffing (Lighthouse CI Server)
+### The Lighthouse CI Server
+
+<img src="https://user-images.githubusercontent.com/39191/68522781-496bad00-0264-11ea-800a-ed86dbb04366.png" width="48%"> <img src="https://user-images.githubusercontent.com/39191/68522269-7917b680-025e-11ea-8d96-2774c0a0b04c.png" width="48%">
+
+Historical reports and advanced report diffing is available with the Lighthouse CI server. For setup of the server itself, see [our server recipe](./recipes/README.md).
 
-Historical reports and advanced report diffing is available on the Lighthouse CI server. For setup of the server itself, see [our server recipe](./recipes/README.md). Once the server is setup, you can use the wizard to create a project.
+Once the server is set up, create a a new project with the lhci wizard:
 
 ```bash
 $ lhci wizard
 ? Which wizard do you want to run? new-project
 ? Which server would you like to use? http://url-of-your-lhci-server.com/
 ? What would you like to name the project? My Favorite Project
 ? Where is the project built? https://travis-ci.org/GoogleChrome/lighthouse-ci
```

Comment:
```suggestion
? Where is the project's code hosted? https://github.com/GoogleChrome/lighthouse-ci
```

Reply:
oh really? i thought this question was asking about your travis URL.

oh you changed it. neat :)